app/view_models/forgot_password_show.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordShow
+  def initialize(params:, session:)
```

Comment:
`params` and `session` only use one key each...what do you think of making those the params?

```ruby
ForgotPasswordShow.new(resend: params[:resend], email: session.delete(:email))
```